frontend/src/app/auth/page.dev.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import React, { Suspense } from 'react';
+import { useSearchParams, redirect, RedirectType } from 'next/navigation';
+import { Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+// this is an npm module, not node:path
+// eslint-disable-next-line  unicorn/prefer-node-protocol
+import path from 'path';
```

Comment:
path is in node. Why the change? 

https://nodejs.org/docs/latest-v18.x/api/path.html#pathnormalizepath

Reply:
This `Redirect` component was copied from the auth webapp - https://github.com/NHSDigital/nhs-notify-iam-webauth/blob/main/src/components/molecules/Redirect/Redirect.tsx

I'm guessing an npm package is being used because it's a client component so doesn't have access to Node core modules.

Reply:
ahhhhh fair!